docs/docs.trychroma.com/pages/reference/js-client.md

```diff
@@ -14,13 +14,13 @@ title: JS Client
 
 Creates a new ChromaClient instance.
 
-#### Parameters
+**Parameters**
```

Comment:
These issues are visible on the site; less so on the GH markdown listing.

Check out https://docs.trychroma.com/reference/js-client to gauge how this will behave.